src/test/kotlin/io/sip3/captain/ce/util/ByteBufUtilKtTest.kt

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright 2018-2019 SIP3.IO, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.sip3.captain.ce.util
+
+import io.netty.buffer.Unpooled
+import org.junit.jupiter.api.Assertions.*
+import org.junit.jupiter.api.Test
+
+class ByteBufUtilKtTest {
+
+    @Test
+    fun `write byte as tlv`() {
+        val byteValue = 1.toByte()
+        val testByteBuf = Unpooled.buffer(4).apply {
+            writeTlv(1, byteValue)
+        }
+
+        assertEquals(4, testByteBuf.capacity())
+
+        assertEquals(1, testByteBuf.readByte().toInt())
```

Comment:
Let's create a companion object with constant `TAG = 1` and use this constant in all tag assertions.